packages/lodestar-beacon-state-transition/src/util/block.ts

```diff
@@ -19,3 +20,18 @@ export function verifyBlockSignature(
     signedBlock.signature.valueOf() as Uint8Array,
   );
 }
+
+export function verifyBlockSignatureFast(
```

Comment:
Can you make this an epochCtx convenience method? Can you move this to `src/fast` subdir?

If you're worried about epochContext.ts getting big, you could still define the method in a separate file.

Reply:
How would that be epoch convinience method?

Reply:
`epochCtx.verifyBlockSignature(state, signedBlock)`
Or we might want to keep fork information in the epochCtx? `epochCtx.verifyBlockSignature(signedBlock)`

Reply:
I guess that's fine, but it seems like clutter of epoch ctx for code that executed exactly in one place :smile: